curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-the-spread-operator-to-evaluate-arrays-in-place.md

```diff
@@ -31,13 +31,13 @@ const maximus = Math.max(...arr);
 `...arr` returns an unpacked array. In other words, it spreads the array. However, the spread operator only works in-place, like in an argument to a function or in an array literal. For example:
 
 ```js
-const spreaded = [...arr];
```

Comment:
This one feels okay as-is to me. While [spreaded isn't generally recognized to be a word, it's at least in common enough usage to warrant a blog post](https://www.grammarly.com/blog/spreaded). And I think there's value in the idea of "once I spread an array, the value has been _spreaded_." Using the nonstandard word here might actually help better convey what's going on.

Reply:
Fixed in https://github.com/freeCodeCamp/freeCodeCamp/pull/51246/commits/4d6f6ac2c79cb7cf674a5f911c204c1deef099a3